.gitignore

```diff
@@ -14,4 +14,5 @@ results
 videos
 saved_approaches
 saved_datasets
-analysis/results
\ No newline at end of file
+analysis/results
+tmp_behavior_states
```

Comment:
note: see uses of "tempfile" in torch_models.py for a usage example

Reply:
Actually instead of creating a temporary directory, maybe we can just use tempfile NamedTemporaryFile and that's it; there's no need for the files to be in any particular directory.

Reply:
sounds good! thanks for this suggestion!

Reply:
i think we need a temp directory, since each `save_checkpoint` call saves two different files

Reply:
i'm not sure how/when to automatically delete these directories though

Reply:
cool. so then `tempfile.TemporaryDirectory` is probably what we want.

Reply:
or maybe `dirpath = tempfile.mkdtemp()`. and btw, we might want to automatically delete these to avoid clogging up our storage. see: https://stackoverflow.com/questions/3223604/how-to-create-a-temporary-directory-and-get-its-path-file-name

Reply:
haha I just saw your comment, GMTA! yeah let's see...